modules/nf-core/sigprofiler/extractor/main.nf

```diff
@@ -0,0 +1,56 @@
+process SIGPROFILER_EXTRACTOR {
+    tag "$meta.id"
+    label 'process_single'
+
+    // TODO nf-core: List required Conda package(s). Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
+    conda "${moduleDir}/environment.yml"
+    container "sigprofilerextractor"
+
+    input:
+    tuple val(meta), path(vcf)
+    val(genome)
```

Comment:
Is the genome the same for every sample in a pipeline run? If not, consider merging into the tuple. Feel free to keep it as-is if the genome is constant.